tests/mocks/repository/lib/mock_mpool.c

```diff
@@ -794,6 +646,22 @@ mpm_mdc_load_file(const char *filename, char **data, int *len)
     return 0;
 }
 
+merr_t
+mpm_mblock_get_base(u64 id, void **data, size_t *wlen)
+{
+    struct mocked_mblock *mb = 0;
+    merr_t                err;
+
+    err = get_mblock(id, &mb);
+    if (err)
+        return err;
+
+    *data = mb->mb_base;
+    *wlen = mb->mb_write_len;
+    return 0;
+}
+
```

Comment:
extra newline

Reply:
New rule:  one extra new line is acceptable.

That being said, I fixed this one.